.github/workflows/integrate.yml

```diff
@@ -7,10 +7,38 @@ on:
 jobs:
   test_pull_request:
     runs-on: ubuntu-latest
+    env:
+      APP_ENV: test
+      APP_PORT: 3000
+      DB_HOST: localhost
+      DB_PORT: 3306
+      DB_NAME: example_db
+      DB_USER: root
+      DB_PASS: example
+      JWT_EXPIRES_IN_SECONDS: 3600
+      JWT_PUBLIC_KEY_PATH: './jwtRS256.key.pub'
+      JWT_PRIVATE_KEY_PATH: './jwtRS256.key'
+
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
           node-version: 12
       - run: npm ci
       - run: npm test
+      - name: Build the docker-compose stack
+        run: docker-compose -f docker-compose.yml up -d mysqldb
+      - name: Check running containers
+        run: docker ps -a
+      - name: Create JWT Public Key
+        run: 'echo "$PUBLIC_KEY" > jwtRS256.key.pub'
```

Comment:
@war1oc CC @Saad-Amjad @asad-mlbd 
BTW for production deployment, how would be supply the JWT key pair to our containers? 
Will be doing something similar like this?

Reply:
This or I was thinking maybe we can try hashing the key and pass as env vars. Just an idea not tested.